### DISCUSSION TITLE
longer release cycles?

### DISCUSSION THREAD

**Discussion by kamil-malinski** [directus/directus#16045] [Archive]
Hi!

Is there a reason why there hasn't been a new release for almost 4 weeks? Before the release cycle was about every 2 weeks.

Thanks!

Kamil

**Comment by rijkvanzanten** [comment]
The release cycle isn't set in stone. We were working on some bigger pieces, so this time around it made more sense to collect a couple more changes and do a bigger release 🤷🏻
